src/components/consul/selectors/ImageSelector.js

```diff
@@ -117,6 +118,10 @@ export const ImageSelector = ({ control, field, imageId, isVolunteer, item }) =>
   };
 
   if (isVolunteer) {
+    const values = jsonParser(value);
+
+    if (!values.length) return null;
```

Comment:
this hides the upload button if there are not files yet :)
i fixed it (for documents as well)